exchanges/okx/okx.go

```diff
@@ -0,0 +1,4847 @@
+		params.Set("type", strconv.Itoa(int(arg.BillType)))
+	}
+	if (arg.BillSubType >= 1 && arg.BillSubType <= 6) || arg.BillSubType == 9 || arg.BillSubType == 11 ||
+		arg.BillSubType == 12 || arg.BillSubType == 14 || (arg.BillSubType >= 100 && arg.BillSubType <= 106) ||
+		(arg.BillSubType >= 110 && arg.BillSubType <= 113) || (arg.BillSubType >= 117 && arg.BillSubType <= 119) ||
+		(arg.BillSubType >= 125 && arg.BillSubType <= 128) || (arg.BillSubType >= 131 && arg.BillSubType <= 132) ||
+		(arg.BillSubType >= 160 && arg.BillSubType <= 162) || (arg.BillSubType >= 170 && arg.BillSubType <= 174) ||
+		(arg.BillSubType >= 200 && arg.BillSubType <= 203) {
+		params.Set("subType", strconv.Itoa(int(arg.BillSubType)))
+	}
+	if arg.After != "" {
+		params.Set("after", arg.After)
+	}
+	if arg.Before != "" {
+		params.Set("before", arg.Before)
+	}
+	if !arg.BeginTime.IsZero() {
+		params.Set("begin", strconv.FormatInt(arg.BeginTime.UnixMilli(), 10))
+	}
+	if !arg.EndTime.IsZero() {
+		params.Set("end", strconv.FormatInt(arg.EndTime.UnixMilli(), 10))
+	}
+	if int(arg.Limit) > 0 {
+		params.Set("limit", strconv.Itoa(int(arg.Limit)))
+	}
+	path := common.EncodeURLValues(route, params)
+	type response struct {
+		Code string                `json:"code"`
+		Msg  string                `json:"msg"`
+		Data []BillsDetailResponse `json:"data"`
+	}
+	var resp response
+	return resp.Data, ok.SendHTTPRequest(ctx, exchange.RestSpot, getBillsDetailsEPL, http.MethodGet, path, nil, &resp, true)
+}
+
+// GetAccountConfiguration retrieve current account configuration.
+func (ok *Okx) GetAccountConfiguration(ctx context.Context) ([]AccountConfigurationResponse, error) {
+	type response struct {
+		Code string                         `json:"code"`
+		Msg  string                         `json:"msg"`
+		Data []AccountConfigurationResponse `json:"data"`
+	}
+	var resp response
+	return resp.Data, ok.SendHTTPRequest(ctx, exchange.RestSpot, getAccountConfigurationEPL, http.MethodGet, accountConfiguration, nil, &resp, true)
+}
+
+// SetPositionMode FUTURES and SWAP support both long/short mode and net mode. In net mode, users can only have positions in one direction; In long/short mode, users can hold positions in long and short directions.
+func (ok *Okx) SetPositionMode(ctx context.Context, positionMode string) (string, error) {
+	if !(positionMode == "long_short_mode" || positionMode == "net_mode") {
+		return "", errors.New("missing position mode, \"long_short_mode\": long/short, only applicable to FUTURES/SWAP \"net_mode\": net")
+	}
+	input := &PositionMode{
+		PositionMode: positionMode,
+	}
+	type response struct {
+		Code string         `json:"code"`
+		Msg  string         `json:"msg"`
+		Data []PositionMode `json:"data"`
+	}
+	var resp response
+	er := ok.SendHTTPRequest(ctx, exchange.RestSpot, setPositionModeEPL, http.MethodPost, accountSetPositionMode, input, &resp, true)
+	if er != nil {
+		return "", er
+	}
+	if len(resp.Data) == 0 {
```

Comment:
```suggestion
	if len(resp.Data) == 1 {
```
Check for the actual response you want. Please do this across all functions that are like this as I'm seeing it a bit.